README.md

```diff
@@ -90,6 +90,20 @@ Currently AWS resources are deployed into a separate Virtual Private Cloud (VPC)
 
 The two VPCs should now be able to communicate with each other. 
 
+## Snapshots
+The cloud resource operator supports the taking of arbitrary snapshots in the AWS provider for both `Postgres` and `Redis`. To take a snapshot you must create a `RedisSnapshot` or `PostgresSnapshot` resource, which should reference the `Redis` or `Postgres` resource you wish to create a snapshot of. The snapshot resource must also exist in the same namespace.
+```
+apiVersion: integreatly.org/v1alpha1
+kind: RedisSnapshot
+metadata:
+  name: my-redis-snapshot
+spec:
+  # The redis resource name for the snapshot you want to take
+  resourceName: my-redis-resource
+
+```  
+*Note* You may experience some downtime in the resource during the creation of the Snapshot
```

Comment:
Is this a reference to the Elasticache documentation around snapshotting?

Does RDS have the same warning around possible impacted service?

Reply:
Tbh, I added that note to remind myself to return to this. I will add links to the documentation around snapshotting and any impact it may have